plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkModifyJob.java

```diff
@@ -127,4 +137,15 @@ protected void canceling() {
       jobThread.interrupt();
     }
   }
+
+  /**
```

Comment:
I didn't do this move as it wasn't clear to me whether all modify jobs would want this.  But it doesn't hurt.